client/src/components/Search/Search.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { SearchWrapper } from './Search.styles';
+
+import { URL_USERS_REQUEST_githubUsers } from '../../URLs/URLs';
+
+import SearchByRepoView from './SearchByRepoView/SearchByRepoView';
+import SearchByUserView from './SearchByUserView/SearchByUserView';
+import SubmitButton from '../UIElements/SubmitButton/SubmitButton';
+import DoubleOptionToggleSwitch from '../UIElements/DoubleOptionToggleSwitch/DoubleOptionToggleSwitch';
+
+const Search = () => {
+    //we can search by user or by repo
+    const [searchType, setSearchType] = useState('user');
+    const [searchGithubNick, setSearchGithubNick] = useState(null);
+    const [searchMinRepoCount, setSearchMinRepoCount] = useState(null);
+    const [searchMaxRepoCount, setSearchMaxRepoCount] = useState(null);
+    const [searchRepoName, setSearchRepoName] = useState(null);
+    const [searchTech, setSearchTech] = useState(null);
+    const [searchMinStarsCount, setSearchMinStarsCount] = useState(null);
+    const [searchMaxStarsCount, setSearchMaxStarsCount] = useState(null);
+
+    const toggleSearchType = () => {
+        if(searchType === 'user') {
+            setSearchType('repository')
+        } else {
+            setSearchType('user')
+        }
+    };
+
+    const saveInputHandler = (event) => {
+        const targetVal = event.target.value;
+        console.log('saveInputHandler', targetVal, event.target.id);
+        switch(event.target.id) {
+            case 'Github nick':
```

Comment:
The IDs should not be human readable. Rather they should be IDs and labels should be taken from somewhere else. I'd change the code like that to be easy to i18nize. Plaease create a task to make the code like that.

Reply:
I added internationalization in todays new commits. Please take a look if I am thinking right